src/ResourceManager/Search/Commands.Management.Search/SearchService/RemoveSearchServiceCommand.cs

```diff
@@ -0,0 +1,92 @@
+
+using Microsoft.Azure.Commands.Management.Search.Models;
+using Microsoft.Azure.Commands.Management.Search.Properties;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Management.Search.SearchService
+{
+    [Cmdlet(VerbsCommon.Remove, SearchServiceNounStr, SupportsShouldProcess = true)]
+    public class RemoveSearchServiceCommand : SearchServiceBaseCmdlet
+    {
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipeline = true,
+            ParameterSetName = InputObjectParameterSetName,
+            HelpMessage = InputObjectHelpMessage)]
+        [ValidateNotNullOrEmpty]
+        public PSSearchService InputObject { get; set; }
+
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            ParameterSetName = ResourceIdParameterSetName,
+            HelpMessage = ResourceIdHelpMessage)]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            ParameterSetName = ResourceNameParameterSetName,
+            HelpMessage = ResourceGroupHelpMessage)]
+        [ResourceGroupCompleter()]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Position = 1,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            ParameterSetName = ResourceNameParameterSetName,
+            HelpMessage = ResourceNameHelpMessage)]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = ForceHelpMessage)]
+        public SwitchParameter Force { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            if (ParameterSetName.Equals(InputObjectParameterSetName, StringComparison.InvariantCulture))
+            {
+                ResourceGroupName = InputObject.ResourceGroupName;
+                Name = InputObject.Name;
+            }
+            else if (ParameterSetName.Equals(ResourceIdParameterSetName, StringComparison.InvariantCulture))
+            {
+                var id = new ResourceIdentifier(ResourceId);
+                ResourceGroupName = id.ResourceGroupName;
+                Name = id.ResourceName;
+            }
+
+            ConfirmAction(Force.IsPresent,
+                string.Format(Resources.RemovingSearchService, Name),
+                string.Format(Resources.RemoveSearchService, Name),
+                Name,
+                () =>
+                {
+                    SearchClient.Services.DeleteWithHttpMessagesAsync(ResourceGroupName, Name).Wait();
```

Comment:
Please add a -PassThru parameter.  When specified, this cmdlet will return "true" to indicate that the operation was successful, or "false" if it was unsuccessful.  If not specified, nothing is returned by the cmdlet.

Reply:
Added -PassThru